proposal.rst

```diff
@@ -273,7 +317,7 @@ Use of Existing Code
 **If you already know about existing code you plan to incorporate or libraries
 you plan to use, please give details.**
 
-FILLME
+I do not exactly know the existing way of transferring data between Letor and Omega, I think it could be fit to do it by Python Pandas library. As I have seen, click models released in C++, so I will do it in C++ too because of speed benefits.
```

Comment:
I'm not convinced you understand the current state of Letor in Xapian and Omega, or what this project would involve. Right now, Omega can record clickstream logs, and there's an analyser that generates training data for Letor. You can then use Xapian's Letor framework to train a model (using one of the rankers we support). One of the things that's needed is to have Omega use Letor to re-rank results.